token-swap/program/src/constraints.rs

```diff
@@ -87,6 +89,18 @@ const VALID_FLAT_CURVES: &[FlatCurve] = &[FlatCurve {
     host_fee_numerator: 20,
     host_fee_denominator: 100,
 }];
+#[cfg(feature = "production")]
+const VALID_STABLE_CURVES: &[StableCurve] = &[StableCurve {
```

Comment:
Let's keep this empty for the moment since I'm not sure if our production users want to allow for a `StableCurve`